terraform-provider-intro/overview.md

```diff
@@ -0,0 +1,52 @@
+In this scenario, you use a Terraform provider to interact with a fictional coffee-shop application, HashiCups, by completing the following actions:
+
+1. Initialize Terraform Workspace
+1. Create a HashiCups order
+1. Update a HashiCups order
+1. Read coffee ingredients (data source)
+1. Delete a HashiCups order
+
+This scenario includes a pre-installed Terraform 0.13, a pre-installed Terraform HashiCups provider and an instance of the HashiCups API running.
+
+### Verify HashiCups API is running
+
+Once you see the HashiCups logs running in your terminal, verify that HashiCups is running.
+
+Click on the following command to send a request to HashiCup's health check endpoint in another terminal.
+
+`curl localhost:19090/health`{{execute T2}} 
+
+### Create HashiCups user
+
+Create a user on HashiCups named `education` with the password `test123`.
+
+`curl -X POST localhost:19090/signup -d '{"username":"education", "password":"test123"}'`{{execute T2}}
+
+This will return the userID, username, and a JWT token. Your JWT authorization token will be used later to retrieve your orders.
+
+```
+{"UserID":1,"Username":"education","token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1OTEwNzgwODUsInVzZXJfaWQiOjIsInVzZXJuYW1lIjoiZWR1Y2F0aW9uIn0.CguceCNILKdjOQ7Gx0u4UAMlOTaH3Dw-fsll2iXDrYU"}
+```
+
+Set `HASHICUPS_TOKEN` to the token you retrieved from invoking the `/signup` endpoint. You will use this later in the tutorial to verify your HashiCups order has been created, updated and deleted.
+
+`export HASHICUPS_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9....`{{execute T2}}
```

Comment:
```suggestion
`export HASHICUPS_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9....`
```
Since they have to copy and paste the token into the command I think include the execute tag might lead people astray.

Reply:
The demo link provided shows a little bit of a different version of this section right now, fwiw.

Reply:
yep! I updated this to ``export HASHICUPS_TOKEN=$(curl -X POST localhost:19090/signin -d '{"username":"education", "password":"test123"}' | jq --raw-output '.token')` so it should work by executing 

I think you were looking at a previous version before Rachel's review